overlord/ifacestate/handlers.go

```diff
@@ -78,6 +79,9 @@ func (m *InterfaceManager) doSetupProfiles(task *state.Task, tomb *tomb.Tomb) er
 	task.State().Lock()
 	defer task.State().Unlock()
 
+	perfTimings := timings.NewForTask(task)
+	defer perfTimings.Save(task.State()) // XXX: should the error from save be returned from handler?
```

Comment:
maybe Save shouldn't return an error, but log direclty and that's it?  we would likely not return the error anyway here but log it

Reply:
Yes, I think that's the right approach, thanks, done.